Sources/PackageDescription/Version.swift

```diff
@@ -96,11 +94,43 @@ public struct Version {
 
 extension Version: Comparable {
     // Although `Comparable` inherits from `Equatable`, it does not provide a new default implementation of `==`, but instead uses `Equatable`'s default synthesised implementation. The compiler-synthesised `==`` is composed of [member-wise comparisons](https://github.com/apple/swift-evolution/blob/main/proposals/0185-synthesize-equatable-hashable.md#implementation-details), which leads to a false `false` when 2 semantic versions differ by only their build metadata identifiers, contradicting SemVer 2.0.0's [comparison rules](https://semver.org/#spec-item-10).
+    
+    /// Returns a Boolean value indicating whether two values are equal.
+    ///
+    /// Equality is the inverse of inequality. For any values `a` and `b`, `a ==
+    /// b` implies that `a != b` is `false`.
+    ///
+    /// This is the default implementation of the equal-to operator (`==`) for
+    /// any type that conforms to `Equatable`.
+    ///
+    /// - Parameters:
+    ///   - lhs: A value to compare.
+    ///   - rhs: Another value to compare.
     @inlinable
     public static func == (lhs: Version, rhs: Version) -> Bool {
         !(lhs < rhs) && !(lhs > rhs)
     }
-
+    
+    /// Returns a Boolean value indicating whether the value of the first
+    /// argument is less than that of the second argument.
+    ///
+    /// This function is the only requirement of the `Comparable` protocol. The
+    /// remainder of the relational operator functions are implemented by the
+    /// standard library for any type that conforms to `Comparable`.
+    ///
+    /// The function determines if the inputs have any items in their
+    /// ``Version/prereleaseIdentifiers`` collections.
+    /// If only one side has prerelease identifiers, then that side will be considered
+    /// to have the lower version number. If they both have prerelease identifiers,
+    /// then the numeric prerelease identifier is considered to be lower version.
+    /// If both prerelease identifiers are numeric, then a numeric comparison is made.
+    ///
+    /// If neither of the parameters have prerelease identifiers, then the <doc://com.apple.documentation/documentation/swift/array/2853083-lexicographicallyprecedes>
+    /// function is used to perform the comparison.
```

Comment:
This description is incorrect. Additionally, it's exposing the implementation detail to the public documentation. I think it's probably better to paraphrase the precedence rules listed on semver.org instead.

Reply:
It might be best if I just wrote

> The precedence is determined according to rules described at the [Semantic Versioning 2.0.0](https://semver.org/#spec-item-11) website.

And leave it at that. I provide the exact link to the precedence rules, and the description there is easy to follow.